packages/graphql/src/schemaTypes/objectTypes/gql-Project.ts

```diff
@@ -98,6 +98,13 @@ export const Project = objectType({
       },
     })
 
+    t.json('savedState', {
+      description: 'Project saved state',
+      resolve: (source, args, ctx) => {
+        return ctx.project.getCurrentProjectSavedState()
```

Comment:
I dug into the CI fails and this is the root of the problem. I didn't go deeper but this should be a good starting point; making this `return null`, running `yarn` to rebuild the schema make the tests pass, locally, when they were otherwise failing.